repos/system_upgrade/el7toel8/actors/targetuserspacecreator/libraries/userspacegen.py

```diff
@@ -54,44 +54,61 @@ def _prep_repository_access(context, target_userspace):
     context.copytree_from('/etc/yum.repos.d', os.path.join(target_userspace, 'etc', 'yum.repos.d'))
 
 
+def _get_expected_product_type(envar_name):
+    """
+    Get expected product type (ga/beta/htb)
+    """
+    # TODO: this function is currently used in another actor as well
+    # - maybe put it into the shared lib? in case we find another use for it..
+    # NOTE: currently we are not printing any debug msg anywhere as LEAPP_DEVEL
+    # envars are already checked
+    val = config.get_env(envar_name, '').lower()
+    return val if val in ('beta', 'htb') else 'ga'
```

Comment:
Stop the execution if there's anything else as a value than these three options.

Reply:
Done. I will add tests for the function yet.